src/openforms/prefill/api/views.py

```diff
@@ -6,6 +6,15 @@
 from rest_framework.views import APIView
 
 from openforms.api.views import ListMixin
+from openforms.registrations.contrib.objects_api.api.serializers import (
+    ObjectsAPIGroupInputSerializer,
+)
+from openforms.registrations.contrib.objects_api.api.views import (
+    OBJECTS_API_GROUP_QUERY_PARAMETER,
+    ObjecttypesListView,
+    ObjecttypeVersionsListView,
+)
```

Comment:
same remarks as above - in this case I would really move these views to `openforms.contrib.objects_api`